StellarApp/Controllers/NewsViewController.swift

```diff
@@ -36,37 +46,53 @@ class NewsViewController: UIViewController {
             }
         }
     }
-    
 }
-
-extension NewsViewController: UICollectionViewDataSource, UICollectionViewDelegate {
+extension NewsViewController: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         return articles.count
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        
         guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "NewsCell", for: indexPath) as? NewsCollectionViewCell else { return UICollectionViewCell() }
         let thisArticle = articles[indexPath.row]
-        cell.articleLabel.text = "Title: \(thisArticle.title)"
-        cell.articleDescription.text = "Description: \(thisArticle.description)"
+        cell.articleLabel.text = thisArticle.source.name
+        cell.articleDescription.text = thisArticle.title
         cell.backgroundColor = .clear
         let url = URL(fileURLWithPath: thisArticle.urlToImage)
         ImageHelper.fetchImageFromNetwork(urlString: thisArticle.urlToImage) { (error, data) in
             if let error = error {
                 print(error.errorMessage())
         }  else if let data = data {
                 cell.articleImage.layer.borderWidth = 5
-                cell.articleImage.layer.borderColor = UIColor.white.cgColor
                 cell.articleImage.layer.cornerRadius = 5
```

Comment:
I think you can remove the border from the image.

Reply:
okay.